app/api/gist/index.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+var Promise = require("bluebird");
```

Comment:
I don't have a source, but I'm pretty sure the Node.js folks have decided to not add native promises. Either way, where `Promise` is the recommended alias (as shown by the Bluebird documentation) I think this is fine. If in the future we need to use native promises for something than I think we can replace the few Bluebird-only API calls.